osu.Game/Screens/Play/RoomSubmittingPlayer.cs

```diff
@@ -19,6 +20,8 @@ public abstract class RoomSubmittingPlayer : SubmittingPlayer
 
         protected readonly PlaylistItem PlaylistItem;
 
+        protected override UserActivity InitialActivity => new UserActivity.InMultiplayerGame(Beatmap.Value.BeatmapInfo, Ruleset.Value);
```

Comment:
Note that this is covering both playlists and multiplayer currently. I'm not sure if you want to expand on this to split the two apart, but it may be best, since playlists is not really "playing with others" or a `MultiplayerGame`.

Reply:
We had this discussion with @bdach already: https://github.com/ppy/osu/pull/14284#discussion_r688935948
Should I take up on your suggestion then?

Reply:
I don't really have any objections to splitting. As long as both are covered somehow.

Reply:
I'm unsure of what status text to use in playlists if `InPlaylistGame` is going to be introduced. Currently this is what I have in mind instead:

Solo — `InSoloGame`
Multi — `InMultiplayerGame`
Playlists - `InSoloGame` (because this is technically a solo play)

Reply:
If we're splitting three-ways then might as well have playlists separate and not inherit solo game, as there may be more info you might want to show (like the playlist name or something).